ansible/roles_ocp_workloads/ocp4_workload_example/.travis.yml

```diff
@@ -0,0 +1,29 @@
+---
+language: python
+python: "2.7"
```

Comment:
2.7?

Reply:
Are we up to travis actually validating these?  If so, I'll update to 2.9